.ebextensions/dbmigrations.config

```diff
@@ -0,0 +1,3 @@
+container_commands:
+  upgrade:
+    command: "python application.py db upgrade"
```

Comment:
I've created the DB using this command, but would be nice to add details of database setup to the README.  (Also... how do you run the tests?  The run_tests.sh script is not happy.)